app/src/main/java/com/babylon/wallet/android/presentation/account/createaccount/CreateAccountViewModel.kt

```diff
@@ -110,7 +104,7 @@ class CreateAccountViewModel @Inject constructor(
                 accessFactorSourcesInput = AccessFactorSourcesInput.ToDerivePublicKey(
                     forNetworkId = args.networkIdToSwitch ?: getProfileUseCase().currentGateway.network.id,
                     factorSource = selectedFactorSource,
-                    isBiometricsProvided = isBiometricsProvided
+                    isBiometricsProvided = true
```

Comment:
the comment "_if main babylon factor source is not present, it will be created during the public key derivation_" can be removed now 🙂 
Since now we do it before we access factor sources!